Common/Source/Draw/DrawFlightMode.cpp

```diff
@@ -13,6 +13,94 @@
 
 extern bool FastZoom; // QUICKDRAW
 
+
+
+LKColor GetUTF8MultimapSymbol(TCHAR* pPict, const int Number)
+{
+LKColor Col = RGB_BLACK;
+  if (pPict ==NULL) return Col;
+    switch(Number)
+    {
+      case 0:
+       _stprintf(pPict, _T("Ⓜ"));
+        Col = LKColor(0,245,255);
+      break;
+
+      case 1:
+        _stprintf(pPict, _T("①"));
+        Col = LKColor(0,255,127);
+      break;
+
+      case 2:
+        _stprintf(pPict, _T("②"));
+        Col = LKColor(205,0,0);
+      break;
+
+      case 3:
+        _stprintf(pPict, _T("③"));
+        Col = LKColor(255,130,71);
+      break;
+
+      case 4:
+        _stprintf(pPict, _T("④"));
+        Col = LKColor(255,165,0);
+      break;
+
+      case 5:
+        _stprintf(pPict, _T("⑤"));
+        Col = LKColor(255,130,71);
+      break;
+
+      case 6:
+        _stprintf(pPict, _T("⑥"));
+        Col = LKColor(82,82,82);
+      break;
+
+      case 7:
+        _stprintf(pPict, _T("⑦"));
+        Col = RGB_DARKBLUE;
+      break;
+
+      case 8:
+        _stprintf(pPict, _T("⑧"));
+        Col = RGB_BLACK;
+      break;
+
+      case 9:
+        _stprintf(pPict, _T("⑨"));
+        Col = RGB_BLACK;
+      break;
+
+      default:
+        _stprintf(pPict, _T("⑩"));
+        Col = RGB_BLACK;
+      break;
+    }
+#ifdef KOBO
+    Col = RGB_LIGHTGREY;
+#endif
+
+    return Col;
+}
+
```

Comment:
string literal containing non usascii characters are not crossplatform safe, neet bo be moved in language file